app/qml/SettingsPanel.qml

```diff
@@ -174,6 +174,24 @@ Page {
                 }
             }
 
+            PanelItem {
+                height: settingsPanel.rowHeight
+                width: parent.width
+                text: qsTr("Accuracy warning")
```

Comment:
What about `Suppress accuracy warning` and keep it off by default? I think it would be more clear what it does

Reply:
I am generally easy. However what about `allow accuracy warning` or similar so all the logic doesn't have to be reversed.

Reply:
@saberraz @wonder-sk

Reply:
I would go with `Show accuracy warnings` and have it on by default...